### PR TITLE
Alt history

### PR DESCRIPTION
**Pull request by Metsanpeitto**
=======

### This is the second time that I ask for a pull request for this same project.
 The first time I encounter a weird problem when pushing the branch to remote: 
  "_Enumerating objects: 26808, done.
Counting objects: 100% (26808/26808), done._"

 Find a workaround took me too many hours and I can't fix the old branch it is too heavy to push it to the remote repository. 
 I was told by a reviewer to don't delete repositories or branches from old pull requests. However, I am not able to fix the old branch. So after talking with my mentor I create a new one and let you know. 

 Here a link to the old review  [""]  -  [https://github.com/Metsanpeitto/Portfolio_1/pull/2](https://github.com/Metsanpeitto/Portfolio_1/pull/2)

#Bellow the old Pull Request message

===================

### This is the final code that I have created :construction: for the Mobile Portfolio project.

- I exported the Figma files with the plugin 'Figma to Html'  :bullettrain_front:  in the same way I do when I design UI / UX.

- Also used SCSS :revolving_hearts: and created a folder structure to easily maintain or/and scale the project

- Includes simple animations :movie_camera: 

=======

⭐️ I hope you can understand. :pray:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
 # Portfolio: setup and mobile version skeleton
```
Comment (sinansevgi) [inline README.md:1]: Kudos for descriptive readme :+1:

File: style.css [hunk 2]
```diff
@@ -252,11 +753,55 @@ ul {
...
+  background-position: center center;
+  background-size: cover;
+}
+
+.image-4 {
+  background: url("../images/v1_239.png");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+}
```
Comment (sinansevgi) [inline style.css]: - [x] Image links are broken. Please fix them.

`..` dots represents the parent directory. But your CSS file and images folder in the same directory.

File: index.html [hunk 3]
```diff
@@ -32,89 +33,379 @@
...
+                    height="20"
+                    width="20"
+                  />
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <div class="l-works">
```
Comment (sinansevgi) [inline index.html]: - [x] You should use semantic HTML tags. You should use the `<section>` tag for the sections of your page.

File: index.html [hunk 4]
```diff
@@ -32,89 +33,379 @@
...
+            </div>
+            <div class="box__action">
+              <button type="button" class="c-b-normal-button">
+                <span class="link">See Project</span>
+              </button>
+            </div>
+          </div>
+        </section>
+      </div>
+
```
Comment (sinansevgi) [inline index.html]: - [x] You should use a list structure for the works section.

Grouping these section with `<li>` and `<ul>` will make your page more semantic, and it gives meaning to the HTML nodes, helping screen readers and other assistive technologies.


```html

<ul>
  <li>Tonic</li>
  <li>Multi-Post Stories </li>
  <li>Skills</li>
</ul>
```

File: index.html [hunk 5]
```diff
@@ -32,89 +33,379 @@
...
+                <div class="dot"></div>
+                <div class="breadcrumb">
+                  <span class="breadcrumb-text">2015</span>
+                </div>
+              </div>
+            </div>
+            <span class="box__parraf"
+              >A daily selection of privately personalized reads; no accounts or
+              sign-ups required.</span
+            >
```
Comment (sinansevgi) [inline index.html]: - [x] You should use `<p>` tag for block-level text elements.

`<span>` tag is an inline container to mark up a part of a text. It shouldn't be used as a block element.

File: index.html [hunk 6]
```diff
@@ -32,89 +33,379 @@
...
```
Comment (sinansevgi) [inline index.html]: - [x] You should use `<p>` tag for block-level text elements.
 
`<span>` tag is an inline container to mark up a part of a text. It shouldn't be used as a block element.

File: index.html [hunk 7]
```diff
@@ -32,89 +33,379 @@
...
```
Comment (sinansevgi) [inline index.html]: - [ ] You should use `<p>` tag for block-level text elements.
 
`<span>` tag is an inline container to mark up a part of a text. It shouldn't be used as a block element.

File: index.html [hunk 8]
```diff
@@ -32,89 +33,379 @@
...
```
Comment (sinansevgi) [inline index.html]: - [ ] You should use `<p>` tag for block-level text elements.
 
`<span>` tag is an inline container to mark up a part of a text. It shouldn't be used as a block element.

File: scss/components/_buttons.scss [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  position: absolute;
+  width: 124px;
+  height: 48px;
+  border: 1px solid $blue-4;
+  box-sizing: border-box;
+  transition: all 0.9s cubic-bezier(0.86, 0, 0.07, 1);
```
Comment (sinansevgi) [inline scss/components/_buttons.scss:18]: - [ ] You should implement the button interactions also (Active, Hover, and Pressed).

Please check [the figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2238) for the proper design.

File: index.html [hunk 10]
```diff
@@ -32,89 +33,379 @@
...
+            <div class="box__bubbles">
+              <div class="bubble"><span class="bubble-text">html</span></div>
+              <div class="bubble"><span class="bubble-text">css</span></div>
+              <div class="bubble">
+                <span class="bubble-text">javaScript</span>
+              </div>
```
Comment (sinansevgi) [inline index.html]: - [ ] The list of project technologies should be a list `<ul>` and each of the technologies should be a list item `<li>`

File: index.html [hunk 11]
```diff
@@ -32,89 +33,379 @@
...
```
Comment (sinansevgi) [inline index.html]: - [x] The list of project technologies should be a list `<ul>` and each of the technologies should be a list item `<li>`

File: index.html [hunk 12]
```diff
@@ -32,89 +33,379 @@
...
```
Comment (sinansevgi) [inline index.html]: - [x] The list of project technologies should be a list `<ul>` and each of the technologies should be a list item `<li>`

File: index.html [hunk 13]
```diff
@@ -32,89 +33,379 @@
...
```
Comment (sinansevgi) [inline index.html]: - [x] The list of project technologies should be a list `<ul>` and each of the technologies should be a list item `<li>`

File: index.html [hunk 14]
```diff
@@ -32,89 +33,379 @@
...
+              <div class="v1_140 language-group">
+                <div class="v1_142 circle orange"></div>
+                <span class="v1_141 language-name">HTML</span>
+              </div>
+              <div class="v1_143 language-group">
+                <div class="v1_145 circle blue"></div>
+                <span class="v1_144 language-name">CSS</span>
+              </div>
+            </div>
+          </div>
```
Comment (sinansevgi) [inline index.html]: - [x] This section is not properly implemented. Images are not visible and styling is missing. Please check the screenshot below to see how it looks on my end. 
Screen Shot 2021-06-05 at 01 22 45

Please check the figma template for proper design.

File: index.html [hunk 15]
```diff
@@ -32,89 +33,379 @@
...
+            <span class="experience-text">Skills</span>
+            <img
+              src="./images/arrow-horizontal.svg"
+              class="arrow-horizontal"
+              alt="bird"
+              height="14"
+              width="8"
+            />
+            <div class="v1_135 line"></div>
+          </div>
```
Comment (sinansevgi) [inline index.html]: - [x] The technology categories ("Languages", "Frameworks" and "Skills") should be in a list `<ul>`, each of them should be in a `<li>` list item.
- [x] The technologies listed under each category should be also list items.

Example structure:
```html

<ul>
  <li>Languages
    <ul>
      <li>JavaScript</li>
      <li>HTML</li>
      <li>CSS</li>
    </ul>
</li>
  <li>Frameworks </li>
  <li>Skills</li>
</ul>
```

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @Metsanpeitto, 

Amazing work with the readme file and the pull request message. :1st_place_medal: 
Kudos for using SCSS :+1:

There are issues that you still need to work on to go further. :arrow_down: 

All professional developers will recommend that new developers start by first learning their fundamentals. This is very astute advice.  If you already have experience, It may seem boring to you. But they all relate as building blocks for almost every project we are going to build.
This project specifically designed to make you familiar with semantic HTML and prepare you for upcoming and more challenging projects. Therefore you should build every part of this project yourself. You shouldn't take the easy way out with the Figma export plugin.

- [ ] You should replace parts you took from the exported code. 

- Exported parts you used in the project made your code hard-to-read with machine-generated class names.
- Exported parts don't follow semantic HTML.


 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Metsanpeitto** [comment]
=======

## The request of code review 3

### This is the third time that I ask for a pull request for this same project.
===

### The first time I encounter a weird problem when pushing the branch to remote, it generated a conflict and I couldn't push the repository :
"Enumerating objects: 26808, done.
Counting objects: 100% (26808/26808), done."

===

### The second time I was pointed by the code reviewer to fix the following errors: 

- Image links are broken. (.. dots represents the parent directory. But your CSS file and images folder in the same directory.)
- Use semantic HTML tags. You should use the <section> tag for the sections of your page.
- You should use a list structure for the works section.
- You should use <p> tag for block-level text elements. (<span> tag is an inline container to mark up a part of a text. It shouldn't be used as a block element.)
- You should implement the button interactions also (Active, Hover, and Pressed).
- The list of project technologies should be a list <ul> and each of the technologies should be a list item <li>
- The 'experience' section is not properly implemented. Images are not visible and styling is missing. 

### Reflections
- All of the errors and suggestions above have been fixed for this pull request. 
- Linter and Stylelint show no errors.
- I search for any other thing that could make my code not good enough and fixed it before asking for this code review. 
- I downloaded the repository and tested it on my end as you would do. I found no errors.
- I didn't know that was not allowed to use Figma plugins to download the design. I understand the consequences and I don't go to do it again.
- In the future, I will allocate more time for possible debugging issues. This is my first week and after Wednesday the pace of the course accelerated in a surprising way. I did have problems catching up in just 2 days.

#Bellow the old Pull Request message

===================

This is the final code that I have created for construction for the Mobile Portfolio project.
I exported the Figma files with the plugin 'Figma to Html' bullettrain_front in the same way I do when I design UI / UX.

-Also used SCSS  and created a folder structure to easily maintain or/and scale the project

-Includes simple animations 

=======

:star: I hope you can understand. :pray:

File: index.html (hunk 3 above)
Comment (DeliceLydia) [reply index.html]: This was well addressed

File: index.html (hunk 4 above)
Comment (DeliceLydia) [reply index.html]: Well addressed this!

File: index.html (hunk 11 above)
Comment (DeliceLydia) [reply index.html]: Well addressed the issue

File: index.html (hunk 12 above)
Comment (DeliceLydia) [reply index.html]: Well addressed this

File: index.html (hunk 13 above)
Comment (DeliceLydia) [reply index.html]: well resolved

File: index.html (hunk 14 above)
Comment (DeliceLydia) [reply index.html]: you fixed this. Good job!

File: index.html (hunk 15 above)
Comment (DeliceLydia) [reply index.html]: Done this as well✔️

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved ✔️

Hi, @Metsanpeitto, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional
- your project section is not centered on the mobile device I suggest that you do it before merging it can make it look good and make sure you correct all typo errors.
### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**